### PR TITLE
Miami fair week

### PR DESCRIPTION
**Pull request by yuki24**
This is the first around for the Miami Fair Week. The contents are currently static, but this allowed me to implement the visual part of the page quickly and will allow the product team to QA and catch mistakes early.

In order to quickly build up the page, I decided to use the [stitch](https://github.com/artsy/stitch) and reaction's Grid component. Also, I couldn't motivate myself to continue to use CoffeeScript, so the new files were written in ES6. Let me know if there's something that could be done better!

screencapture-localhost-5000-miami-fair-week-1508532321807

### CODE REVIEW

File: desktop/apps/miami_fair_week/components/MiamiFairWeekPage.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            <Col lg={7} md={7} sm={6} xs={12}>
+              <img style={{ marginBottom: 10, width: '100%' }} src='https://d7hftxdivxxvm.cloudfront.net/?resize_to=width&src=https%3A%2F%2Fartsy-media-uploads.s3.amazonaws.com%2FOX8QZ5TCXVt8szczr7oWZQ%252Fammann.jpg&width=1100&quality=95' />
+            </Col>
+            <Col lg={5} md={5} sm={6} xs={12}>
+              <Title titleSize='small' style={{ margin: '0 0 5px', lineHeight: 1 }}>
+                The 10 Best Booths at Design Miami/
+              </Title>
+              <Text textStyle='primary' textSize='small'>
+                Artsy Editorial
+              </Text>
```
Comment (damassi) [inline desktop/apps/miami_fair_week/components/MiamiFairWeekPage.js]: I'm a big fan of this approach -- sweeping over with static content and then iterating / componentizing (only) when necessary. It's often the case where breaking things apart from the get-go leads to premature optimization.

File: desktop/apps/miami_fair_week/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+      res.send(layout)
+    } catch (error) {
+      next(error)
+    }
+  }
+}
+
+// This smells... For some reason 'export default' doesn't work when this file is required by a coffeescript file.
+module.exports = new EditableMiamFairWeekPage({
```
Comment (damassi) [inline desktop/apps/miami_fair_week/index.js]: Ah yeah, this is confusing. From CoffeeScript you need to 
```coffeescript
foo = require('./some-es6-file').default
```

File: desktop/components/json_page/es6.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import express from 'express'
+import JsonPage from './index'
+import routes from './routes'
+
+export default class JSONPage {
```
Comment (craigspaeth) [inline desktop/components/json_page/es6.js]: 👍  nice es6 wrapper!

**Comment by craigspaeth** [comment]
Love this! Great example of how we can do staticish pages in Force with all the new tools. Well done!

**Comment by alloy** [comment]
Nice job @yuki24 👌 And very happy to see a big component rather than an early refactored one 👏

**Comment by yuki24** [comment]
merging this PR for visual QA.

**Pull request merged**